Comparison Engine/Comparison Engine/Form1.cs

```diff
@@ -60,21 +62,21 @@ private void initializeList()
 
         private void populateLists()
         {
-            this.bars = Data.GetBars();
-            this.drinks = Data.GetDrinks();
+            barManager.barDictionary = Data.GetBars();
+            drinkManager.drinkDictionary = Data.GetDrinks();
         }
```

Comment:
I'm starting to think that this could be moved from Form1 as it definitely doesn't need to be in this class. Form1 could do with just the barManager and drinkManager references. Probably could move this to Data as well

Reply:
Probably a good idea. However, would'nt it be better to initialize dictionaries in controller class immediately (public Dictionary<int, Bar> barDictionary = Data.GetBars())?

Reply:
if that works, definitely